src/python/zquantum/core/openfermion/_qiskit_conversions.py

```diff
@@ -0,0 +1,88 @@
+############################################################################
+#   Copyright 2017 Rigetti Computing, Inc.
+#   Modified by Zapata Computing 2020 to work for qiskit's WeightedPauliOperator.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+############################################################################
+
+"""
+Translates OpenFermion Objects to qiskit WeightedPauliOperator objects
+"""
+from openfermion import QubitOperator, count_qubits
+from typing import Union
+from qiskit.aqua.operators import WeightedPauliOperator
+from qiskit.quantum_info import Pauli
+
+
+def qubitop_to_qiskitpauli(qubit_operator: QubitOperator) -> WeightedPauliOperator:
+    """
+    Convert a OpenFermion QubitOperator to a WeightedPauliOperator.
+
+    Args:
+        qubit_operator: OpenFermion QubitOperator to convert to a qiskit.aqua.operators.WeightedPauliOperator
+
+    Returns:
+        WeightedPauliOperator representing the qubit operator
+    """
+    if not isinstance(qubit_operator, QubitOperator):
+        raise TypeError("qubit_operator must be a OpenFermion " "QubitOperator object")
+
+    if qubit_operator == QubitOperator():
```

Comment:
I think the conversion to qiskit won't behave as intended because `QubitOperator()` yields the zero operator, not the identity operator. The identity operator would be `QubitOperator(())`. Also, should we be using the same "I" representation for the identity term of an operator that also contains other terms?

Reply:
Fixed here: https://github.com/zapatacomputing/z-quantum-core/pull/222/commits/906d53e4f340993f4d9193f8c061281fbf093b85

Reply:
If I'm not mistaken, the check for an empty operator isn't necessary since the rest of the code would still return the correct value.

Reply:
True! Fixed here: https://github.com/zapatacomputing/z-quantum-core/pull/222/commits/906d53e4f340993f4d9193f8c061281fbf093b85

Reply:
But still left the test, just in case ;)